### ISSUE TITLE
[UnexpectedToken]: Transform function returned a code string got a BABEL_PARSER_SYNTAX_ERROR

### ISSUE THREAD

**Issue by emosheeep** [sxzz/unplugin-ast#25]
### Describe the bug

Recently I tried to use this plugin to replace the the `lodash` used by dependencies  with `lodash-es` all over the codebase. I tried to return an AST Node before, which would be wrapped with parentheses and you'v fixed it in https://github.com/sxzz/unplugin-ast/issues/22, then I tried to return the code string directly, which still failed. Could you please help me out?

I have made a reproduction at [codesandbox](https://codesandbox.io/s/unplugin-ast-bug-reproduction-w4np5f?file=/vue.config.js)

```js
WebpackAstPlugin({
  include: [/\.[jt]sx?$/],
  transformer: [
    ({
      onNode: node => node.type === 'VariableDeclaration'
        && node.declarations?.[0].type === 'VariableDeclarator'
        && node.declarations[0].init?.type === 'CallExpression'
        && node.declarations[0].init.callee?.name === 'require'
        && node.declarations[0].init.arguments[0]?.type === 'StringLiteral'
        && /lodash\/\w+/.test(node.declarations[0].init.arguments[0]?.value),
      transform(node) {
        const originNode = node.declarations[0].init;
        const importer = originNode.arguments[0].value;
        const importSource = importer.replace('lodash', 'lodash-es');
        const code = `var ${node.declarations[0].id.name} = require('${importSource}').default;`;
        console.log(code); // var isNull = require('lodash-es/isNull').default;
        return code;
      },
    }),
  ],
})
```

<img alt="image" src="https://user-images.githubusercontent.com/31957758/203340920-b114789f-4930-427f-930a-3455fa5f9b17.png">

### Reproduction

https://codesandbox.io/s/unplugin-ast-bug-reproduction-w4np5f?file=/vue.config.js

### System Info

```Shell
System:
    OS: macOS 13.0.1
    CPU: (8) arm64 Apple M2
    Memory: 79.88 MB / 8.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 18.11.0 - ~/.nvm/versions/node/v18.11.0/bin/node
    Yarn: 1.22.19 - ~/Library/pnpm/yarn
    npm: 8.19.2 - ~/.nvm/versions/node/v18.11.0/bin/npm
  Browsers:
    Chrome: 107.0.5304.110
    Safari: 16.1
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/sxzz/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/sxzz/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by sxzz** [comment]
Fixed in `unplugin-ast@0.5.7`.

**Comment by emosheeep** [comment]
So fast🤣
